cmd/remote_enforcer.go

```diff
@@ -0,0 +1,302 @@
+package main
+
+import (
+	"container/list"
+	"crypto/ecdsa"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"os/user"
+	"strconv"
+	"sync"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/aporeto-inc/trireme/cache"
+	"github.com/aporeto-inc/trireme/collector"
+	"github.com/aporeto-inc/trireme/enforcer"
+	_ "github.com/aporeto-inc/trireme/enforcer/utils/nsenter"
+	"github.com/aporeto-inc/trireme/enforcer/utils/packet"
+	"github.com/aporeto-inc/trireme/enforcer/utils/rpcwrapper"
+	"github.com/aporeto-inc/trireme/enforcer/utils/tokens"
+	"github.com/aporeto-inc/trireme/policy"
+	"github.com/aporeto-inc/trireme/supervisor"
+	"github.com/aporeto-inc/trireme/supervisor/iptablesutils"
+	"github.com/aporeto-inc/trireme/supervisor/provider"
+)
+
+const (
+	ipcProtocol         = "unix"
+	defaultPath         = "/var/run/default.sock"
+	statsContextID      = "UNUSED"
+	defaulttimeInterval = 2
+	envStatsChannelPath = "STATSCHANNEL_PATH"
+	envSocketPath       = "SOCKET_PATH"
+)
+
+//CollectorImpl exported
+type CollectorImpl struct {
+	Flowentries *list.List
+	sync.RWMutex
+}
+type collectorentry struct {
+	L4FlowHash string
+	entry      *enforcer.StatsPayload
+}
+
+//CollectFlowEvent expoted
+func (c *CollectorImpl) CollectFlowEvent(contextID string, tags policy.TagsMap, action string, mode string, sourceID string, tcpPacket *packet.Packet) {
+	l4FlowHash := tcpPacket.L4FlowHash()
```

Comment:
[Style] Missing/Extra newline at beginning and end of functions and intermediate sections. Follow the norms in the file you are editing. While creating your own be consistent with linespacing.